src/main/resources/application.yml

```diff
@@ -83,3 +83,9 @@ oauth:
 policy:
   minimum-booking-notice-period-days: 2
   maximum-booking-notice-period-days: 28
+
+task:
+  expired-visit:
+    enabled: true
+    cron: 0 0 * * * *
```

Comment:
runs on the hour, so booking could be up to 1:19:59 old

Reply:
Running on the hour would be a good starting point to see how often this edge case occurs, this value will be refined from alpha.